### PR TITLE
Evo 1439 mark edited posts and comments as edited

### LINKED ISSUE #1439: Mark edited posts and comments as "edited"

### PR DESCRIPTION
**Pull request by KevinTriplett**
Closes #1439 

This PR is dependent on [Hylo-Node PR 999](https://github.com/Hylozoic/hylo-node/pull/999)

### CODE REVIEW

**Comment by KevinTriplett** [comment]
One thing I haven't tested is how the 'Edited' text looks with proposals. For some reason, I'm getting an error on my dev machine when trying to create a proposal post.

**Comment by KevinTriplett** [comment]
Just realized that I need to add translation for Spanish

File: src/components/PostCard/PostDetails/PostDetails.scss [hunk 1]
```diff
@@ -73,6 +73,19 @@
   }
 }
 
+.timestamp {
+  // KMT: composes seems to be ignored, need help to understand what I'm doing wrong
```
Comment (thomasgwatson) [inline src/components/PostCard/PostDetails/PostDetails.scss:77]: Ignored or over-written? I don't have a lot of experience using the `composes` keyword either

**Review by thomasgwatson — APPROVED** [review]
lgtm

**Pull request merged**